.github/workflows/comment-docs-download.yml

```diff
@@ -0,0 +1,43 @@
+name: Comment docs on pull request
```

Comment:
Instead of making two separate extra comments for wheels & docs, I would prefer either having everything in a single comment, or even better as a link added at the bottom of the first PR comment (like we did for the documentation preview before)

Reply:
Okay. Links added below the initial comment like

📖 Download documentation
⚙️ Download wheels

would be cool.